integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/ViewPopRedirectTests/RedirectTest.java

```diff
@@ -120,12 +120,12 @@ public void testRedirect() throws Exception {
                 "but it is active.");
 
         //Call popAndRedirect function to pass the message to REDIRECT_STORE
+        log.info("=== Popping Message from Queue and Redirecting ===");
         messageProcessorClient.popAndRedirectMessage(PROCESSOR_NAME, REDIRECT_STORE_NAME);
 
-
         //Call the getMessage function from the REDIRECT_PROCESSOR
         String returnedMessage = messageProcessorClient.browseMessage(REDIRECT_PROCESSOR_NAME);
-        System.out.println("=== RETURNED MESSAGE === \n" + returnedMessage);
+        log.info("=== RETURNED MESSAGE === \n" + returnedMessage);
```

Comment:
We don't need these logs IMO as we are asserting them

Reply:
I will remove them